CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Changed
 
  - Update PR template [#194](https://github.com/ios-course/ironfoudation-team-project/pull/194)
+- Add `SitterProfileView` to display and edit personal information about the sitter - [#190](https://github.com/ios-course/ironfoudation-team-project/pull/190)
```

Comment:
Please move your changes to `Added` section.

Reply:
Sorry, didn't check after resolving the merge conflict.
Fixed.